### PR TITLE
Combobox

### PR DESCRIPTION
**Pull request by ericanaughton**
:cactus: 

ready to go

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,8 +8,15 @@
     <script src="js/jquery-scrolltofixed-min.js" type="text/javascript"></script>
     <link rel='stylesheet' href='js/jquery-ui-1.11.4.custom/jquery-ui.min.css' />
     <script type="text/javascript" src="js/stickynav.js"></script>
+    <link rel="stylesheet" href="//code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css">
+    <script src="//code.jquery.com/jquery-1.10.2.js"></script>
+    <script src="//code.jquery.com/ui/1.11.4/jquery-ui.js"></script>
+    <script src='js/jquery-1.11.3.min.js'></script>
+    <script src='js/jquery-ui-1.11.4.custom/external/jquery/jquery.js'></script>
+    <script src='js/jquery-ui-1.11.4.custom/jquery-ui.min.js'></script>
```
Comment (ahaynes-ador) [inline index.html]: You've got multiple versions and duplications of JQuery and JQuery UI here.

File: index.html [hunk 2]
```diff
@@ -25,54 +31,44 @@
             <p>Ayudando a los Hispanohablantes a Aprender Terminolog&iacute;a de Desarollo Web</p>
         </div>
     </header>
-    <div class="nav-placeholder" style="height: 42px;">
+<h4 class='searchlabel'>Select an element to view definition, memorization aids, audio pronunciation, and code sample:<br>Selecione un elemento para ver la definici&oacute;n, ayuda para recordar, la pronunciaci&oacute;n con audio, y ejemplos de como usar la etiqueta:</h4>
+
+<div class="sticky-combo">
         <nav class="">
             <div class="nav-inner">
                 <div class="nav-inner-most">
```
Comment (ahaynes-ador) [inline index.html]: Why the deep nesting here?

File: index.html [hunk 3]
```diff
@@ -25,54 +31,44 @@
...
-    <script src='js/jquery-ui-1.11.4.custom/jquery-ui.min.js'></script>
-    <script src='js/scripts.js'></script>
+<main>
+  <div class="main-inside inside" id="main">
+  </div>
+</main>
+  <footer class="site-footer">
+    Project by Team Open Brace, From <a href="http://ywebca.org/" target="_blank">YWebCA</a> Summer 2015.<br>
+    Download the template HTML file for making your own card <a href="elements/template.html" download>here</a>.<br>
+    Check out our <a href="https://github.com/kevinbrianfahy/LMHT" target="_blank">GitHub repository</a> to send a pull request for your card!
```
Comment (ahaynes-ador) [inline index.html:56]: I think it would be more semantic to use `<p>`s here rather than `<br>`s. You can get rid of the margin with CSS if you don't like it.

File: css/style.css [hunk 4]
```diff
@@ -23,47 +20,58 @@ nav {
...
 }
 
 .fixed .nav-inner-most {
-    width: 100%;
+  width: 100%;
 }
 
-nav ul {
+  nav ul {
     margin: 0;
```
Comment (itsthatguy) [inline css/style.css]: Can you normalize your indentation strategy? you're using 2 spaces, and 4 spaces. ideally it should all be 2 spaces.

File: index.html [hunk 5]
```diff
@@ -1,6 +1,5 @@
 <!DOCTYPE HTML>
 <html>
-
 <head>
     <meta charset="UTF-8">
```
Comment (itsthatguy) [inline index.html]: 2 space indentation

File: js/scripts.js [hunk 6]
```diff
@@ -27,7 +34,7 @@ var app = {
       }
 
     });
-
+*/
```
Comment (ahaynes-ador) [inline js/scripts.js]: Is there a reason this is commented out and not deleted?

File: js/scripts.js [hunk 7]
```diff
@@ -72,7 +79,7 @@ var app = {
       $("option[value='" + url + "']").prop("selected", true);
 
       // Refresh jQuery UI menu
-      $('#tags').selectmenu("refresh");
+      //$('#tags').selectmenu("refresh");
```
Comment (ahaynes-ador) [inline js/scripts.js]: Is there a reason this is commented out and not deleted?

File: index.html (hunk 1 above)
Comment (ericanaughton) [inline index.html]: which ones are duplicated? im not seeing it

File: index.html (hunk 2 above)
Comment (ericanaughton) [inline index.html]: this how savance wrote the sticky nav code, i need to fix the indentation. i always use 2 spaces, but i failed to edit what was already there

File: index.html (hunk 3 above)
Comment (ericanaughton) [inline index.html:56]: i agree, but we're getting rid of that footer pretty soon anyway sooo...

File: js/scripts.js (hunk 6 above)
Comment (ericanaughton) [inline js/scripts.js]: it should be deleted.

File: js/scripts.js (hunk 7 above)
Comment (ericanaughton) [inline js/scripts.js]: it should be deleted.

File: css/style.css (hunk 4 above)
Comment (ericanaughton) [inline css/style.css]: i will communicate that to savance. (he uses 4, i use 2). i should have edited those anyway.

File: index.html (hunk 1 above)
Comment (ahaynes-ador) [inline index.html]: Redundant and differing versions:

```
<script src="//code.jquery.com/jquery-1.10.2.js"></script>
<script src='js/jquery-1.11.3.min.js'></script>
<script src='js/jquery-ui-1.11.4.custom/external/jquery/jquery.js'></script>
```

```
<script src="//code.jquery.com/ui/1.11.4/jquery-ui.js"></script>
<script src='js/jquery-ui-1.11.4.custom/jquery-ui.min.js'></script>
```

File: index.html (hunk 2 above)
Comment (ahaynes-ador) [inline index.html]: Not the indentation, the depth of the nested elements. `div -> nav -> div -> div -> ul`

Comment (ericanaughton) [inline index.html]: @SaVanceX could you talk about your sticky nav here?

File: css/style.css (hunk 4 above)
Comment (ericanaughton) [inline css/style.css]: also the previous group used 4 spaces a lot, so in the future we will be mindful of editing those indentations before making a PR

File: index.html (hunk 1 above)
Comment (ericanaughton) [inline index.html]: some of those jquery references were for the droplist.. not exactly sure which ones. ill look into that.

File: index.html (hunk 2 above)
Comment (savance-ford) [inline index.html]: @ahaynes-gpsw so i basically taught myself how to do this..
I basically followed along with the guy teaching how to do this sticky bar:+1:

Comment (ahaynes-ador) [inline index.html]: Looks like a good opportunity to deepen your understanding by refactoring it down to the minimum depth. :wink:

Comment (savance-ford) [inline index.html]: @ahaynes-gpsw so this is the link to the tut I followed. https://www.youtube.com/watch?v=utonytGKodc
when I remove a div or 2 I get the same problems I had before when the nav would jump out of place as well as the body. This is the best way I was able to solve that problem :) I can mos def check out something that would not need so much nesting. (im still pretty new to this, still learning)

File: index.html [hunk 8]
```diff
@@ -1,78 +1,72 @@
...
+  <meta charset="UTF-8">
+  <title>LMHT</title>
+  <link type='text/css' href='css/style.css' rel='stylesheet' />
+  <link rel='stylesheet' href='js/jquery-ui-1.11.4.custom/jquery-ui.min.css' />
+  <link rel="stylesheet" href="//code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js" type="text/javascript"></script>
+  <script type="text/javascript" src="js/stickynav.js"></script>
+  <script src="//code.jquery.com/ui/1.11.4/jquery-ui.js"></script>
+  <script src='js/jquery-ui-1.11.4.custom/external/jquery/jquery.js'></script>
+  <script src='js/jquery-ui-1.11.4.custom/jquery-ui.min.js'></script>
```
Comment (ahaynes-ador) [inline index.html:9]: You've still got duplicates:

```
<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js" type="text/javascript"></script>
<script src='js/jquery-ui-1.11.4.custom/external/jquery/jquery.js'></script>
```

```
<script src="//code.jquery.com/ui/1.11.4/jquery-ui.js"></script>
<script src='js/jquery-ui-1.11.4.custom/jquery-ui.min.js'></script>
```

I also wonder if `<link rel="stylesheet" href="//code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css">` includes the base CSS or if it's just overrides for `<link rel='stylesheet' href='js/jquery-ui-1.11.4.custom/jquery-ui.min.css' />`. Worth investigating.

Comment (ericanaughton) [inline index.html:9]: k ill check it out

File: index.html [hunk 9]
```diff
@@ -1,78 +1,72 @@
...
-        <div class="main-inside inside" id="main">
+  </header>
+  <div>
+    <h4 class='searchlabel'>Select an element to view definition, memorization aids, audio pronunciation, and code sample:<br>Selecione un elemento para ver la definici&oacute;n, ayuda para recordar, la pronunciaci&oacute;n con audio, y ejemplos de como usar la etiqueta:</h4>
+  </div>
+  <div class="sticky-combo">
+    <nav class="">
+      <div class="nav-inner">
+        <div class="nav-inner-most">
+          <ul class="clearfix">
```
Comment (ahaynes-ador) [inline index.html]: This is still super nested and I don't immediately see the purpose. Also, see my other comment.

At the very least, you don't need `class=""` on that `nav`.

File: js/stickynav.js [hunk 10]
```diff
@@ -1,46 +1,46 @@
...
+
   // define variables
   var navOffset, scrollPos = 0;
-  
+
   // add utility wrapper elements for positioning
-  $("nav").wrap('<div class="nav-placeholder"></div>');
+  $("nav").wrap('<div class="sticky-combo"></div>');
   $("nav").wrapInner('<div class="nav-inner"></div>');
   $(".nav-inner").wrapInner('<div class="nav-inner-most"></div>');
```
Comment (ahaynes-ador) [inline js/stickynav.js:10]: These wrappers duplicate the excessive nesting hard coded into the HTML file, so it's actually rendering this:

screen shot 2015-11-24 at 12 14 21 pm

File: index.html [hunk 11]
```diff
@@ -1,78 +1,59 @@
...
-    <script src="js/jquery-scrolltofixed-min.js" type="text/javascript"></script>
-    <link rel='stylesheet' href='js/jquery-ui-1.11.4.custom/jquery-ui.min.css' />
-    <script type="text/javascript" src="js/stickynav.js"></script>
+  <meta charset="UTF-8">
+  <title>LMHT</title>
+  <link type='text/css' href='css/style.css' rel='stylesheet' />
+  <link rel="stylesheet" href="//code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js" type="text/javascript"></script>
+  <script type="text/javascript" src="js/stickynav.js"></script>
+  <script src='js/jquery-ui-1.11.4.custom/jquery-ui.min.js'></script>
```
Comment (ahaynes-ador) [inline index.html:9]: I would swap line 9 and 10, just to be safe.

**Comment by ahaynes-ador** [comment]
Other than that tiny nit, all :+1: . :sunglasses:
